lib/features/upload/domain/usecases/local_image_picker_interactor.dart

```diff
@@ -0,0 +1,34 @@
+import 'package:core/presentation/state/failure.dart';
+import 'package:core/presentation/state/success.dart';
+import 'package:core/utils/platform_info.dart';
+import 'package:dartz/dartz.dart';
+import 'package:file_picker/file_picker.dart';
+import 'package:tmail_ui_user/features/upload/domain/exceptions/pick_file_exception.dart';
+import 'package:tmail_ui_user/features/upload/domain/extensions/platform_file_extension.dart';
+import 'package:tmail_ui_user/features/upload/domain/state/local_image_picker_state.dart';
+
+class LocalImagePickerInteractor {
+
+  LocalImagePickerInteractor();
+
+  Stream<Either<Failure, Success>> execute() async* {
+    try {
+      yield Right<Failure, Success>(LocalImagePickerLoading());
+
+      final filePickerResult = await FilePicker.platform.pickFiles(
+        type: FileType.image,
+        allowMultiple: false,
+        withReadStream: PlatformInfo.isWeb
```

Comment:
why not `true` for all platform?

Reply:
On mobile we use path files. We don't have to transmit bytes through the extra header dio request